chart/openfaas/templates/controller-rbac.yaml

```diff
@@ -179,12 +212,12 @@ metadata:
     component: faas-controller
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
-  name: {{ .Release.Name }}-controller
-  namespace: {{ $functionNs | quote }}
+  name: {{ .Release.Name }}-profiles
```

Comment:
note that this diff is misleading, i have only added a new Role and RoleBinding, this is just groupnig the changes in an odd way to make it look like i edited the existing RoleBinding

Reply:
Thanks for that note 👍